addons/account/models/account_move.py

```diff
@@ -2926,6 +2933,8 @@ def _compute_hash(self, previous_hash):
         hash_string = sha256((previous_hash + self.string_to_hash).encode('utf-8'))
         return hash_string.hexdigest()
 
+    @api.depends(_get_integrity_hash_fields)
```

Comment:
```suggestion
    @api.depends(lambda self: self._get_integrity_hash_fields())
```

Otherwise you can't extend that function in other modules.
I just hope that you won't have a linter error :see_no_evil: 
https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/unnecessary-lambda.html